### PR TITLE
fix: crash when the arr argument for concatArr isn't specified

### PR DESCRIPTION
**Pull request by rubyowo**

### CODE REVIEW

**Comment by rubyowo** [comment]
https://github.com/ZeppelinBot/CLA/pull/29

File: backend/src/templateFormatter.ts [hunk 1]
```diff
@@ -346,6 +346,7 @@ const baseValues = {
     return [...args].join("");
   },
   concatArr(arr, separator = "") {
+    if(!arr) return "";
```
Comment (almeidx) [inline backend/src/templateFormatter.ts:349]:
```suggestion
    if (!Array.isArray(arr)) return "";
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: backend/src/templateFormatter.ts (hunk 1 above)
Comment (rubyowo) [reply backend/src/templateFormatter.ts:349]: thx

File: backend/src/templateFormatter.ts [hunk 2]
```diff
@@ -346,6 +346,7 @@ const baseValues = {
     return [...args].join("");
   },
   concatArr(arr, separator = "") {
+    if(!Array.isArray(arr)) return "";
```
Comment (almeidx) [inline backend/src/templateFormatter.ts:349]:
```suggestion
    if (!Array.isArray(arr)) return "";
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Comment by Dragory** [comment]
@cla-bot check

**Pull request merged**